Apps/AdminWebClient/src/ClientApp/src/views/Feedback.vue

```diff
@@ -168,5 +220,82 @@ export default class FeedbackView extends Vue {
                 this.isLoading = false;
             });
     }
+
+    private onTagFocus(feedback: UserFeedback) {
+        this.focusedTags = feedback.tags;
+    }
+
+    private onTagChange(
+        input: (string | AdminTag)[],
+        feedbackItem: UserFeedback
+    ) {
+        this.$nextTick(() => {
+            // Reset the feedback tags until the backend updates it
+            feedbackItem.tags = this.focusedTags;
+
+            const lastTag = input[input.length - 1];
+
+            var newTag: string | AdminTag = "";
+
+            // Look for the existing tags for a name match
+            if (typeof lastTag === "string") {
+                var foundIndex = this.adminTags.findIndex(
+                    (x) => x.name === lastTag
+                );
+                if (foundIndex > 0) {
+                    newTag = this.adminTags[foundIndex];
+                } else {
+                    newTag = lastTag;
+                }
+            } else {
+                newTag = lastTag;
+            }
+
+            if (typeof newTag === "string") {
+                this.isLoadingTag = true;
+                this.userFeedbackService
+                    .createTag(feedbackItem.id, newTag)
+                    .then((newTag) => {
+                        feedbackItem.tags.push(newTag);
+                        this.adminTags.push(newTag);
+                    })
+                    .catch((err) => {
+                        this.showFeedback = true;
+                        this.bannerFeedback = {
+                            type: ResultType.Error,
+                            title: "Error",
+                            message: "Error creating tag",
+                        };
+                        console.log(err);
+                    })
+                    .finally(() => {
+                        this.isLoadingTag = false;
+                    });
+            } else {
+                this.isLoadingTag = true;
+                this.userFeedbackService
+                    .addTag(feedbackItem.id, newTag)
+                    .then((newTag) => {
+                        feedbackItem.tags.push(newTag);
+                    })
+                    .catch((err) => {
+                        this.showFeedback = true;
+                        this.bannerFeedback = {
+                            type: ResultType.Error,
+                            title: "Error",
+                            message: "Error adding tag",
+                        };
+                        console.log(err);
+                    })
+                    .finally(() => {
+                        this.isLoadingTag = false;
+                    });
+            }
+        });
+    }
+
+    private filter(item: AdminTag, queryText: string): boolean {
```

Comment:
Its nice of you to want to implement everything in one shot but we created another user story to handle this, as a matter of fact I don't know if we want to use a partial search (textbox) or would that be a exact match (combobox) when filtering tags, also notice that we do have case sensitive tags but your filter doesn't take that in consideration

Reply:
Part of the nice-ness of using v-combobox is that it allows search and filter pretty much out of the box. As it is it pretty much does what we need to do to complete that story.

Regarding using a textbox vs combobox that is an implementation detail that was not discussed during design.

We can check with Nino about the case-sensitive search, if you read the method it accounts for the case.